vendor/github.com/mongodb/mongo-tools-common/testutil/testutil.go

```diff
@@ -167,3 +168,12 @@ func MergeOplogStreams(input [][]db.Oplog) []db.Oplog {
 
 	return ops
 }
+
+// CountCursorResults counts number of results in a cursor.
+func CountCursorResults(c *mongo.Cursor) int {
```

Comment:
Could we use [profileCollection.CountDocuments()](https://godoc.org/go.mongodb.org/mongo-driver/mongo#Collection.CountDocuments) instead? It seems like we don't need this function.

Reply:
Sure, didn't know that existed, thanks!

Reply:
oh, actually I'm wrong. That counts the documents in the collection, by functions counts the documents returned by a query :(

Reply:
@pmeredit yes, you'll still have to pass a filter into CountDocuments(). But with the filter it should work.